ngraph/core/include/openvino/opsets/opset.hpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <locale>
+#include <map>
+#include <mutex>
+#include <set>
+#include <utility>
+
+#include "ngraph/factory.hpp"
```

Comment:
will it changed later?

Reply:
Yes we will remove it with old API

Reply:
Do you mean we will remove it before 2022.1 is released?